README.md

```diff
@@ -1,7 +1,116 @@
 python-gp
 ===========
+[![Python3.6](https://img.shields.io/badge/python-3.6-red.svg)](https://www.python.org/downloads/release/python-369/)
+[![Python3.7](https://img.shields.io/badge/python-3.7-red.svg)](https://www.python.org/)
+[![Python3.8](https://img.shields.io/badge/python-3.8-red.svg)](https://www.python.org/)
+[![GPL license](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/old-licenses/gpl-3.0.html)
+
+Cartesian Genetic Programming (CGP) in Python.
+
+This library implements Cartesian Genetic Programming in pure Python. It provides Python data structures to represent and evolve a two-dimensional directed graph (the genotype) that can be translated into a computational graph (the phenotype) implementing a particular mathematical expression. The current implementation uses an evolutionary algorithm,  (mu + lambda) evolution strategies, to evolve a population of Cartesian graphs in order to optimize an objective function.
+<img src="cgp-sketch.png" alt="CGP Sketch" width="600"/>
+
+Basic usage
+===========
+
+A simple example of CGP applied to a symbolic regression problem can be found in `examples/example_evo_regression.py`.
+
+The basic steps are:
+
+1. Define an objective function. 
+
+   The objective function needs to take an individual as input variable and update the `fitness` of the individual.
+```python
+def objective(individual):
+      """Objective function of the regression task.
+
+       Parameters
+      ----------
+      individual : gp.CGPIndividual
+          Individual of the Cartesian Genetic Programming Framework.
+
+      Returns
+      -------
+      gp.CGPIndividual
+          Modified individual with updated fitness value.
+      """
+      # Compute the fitness value
+      fitness = ...
+	  individual.fitness = fitness
+	  return individual
+```
+2. Define parameters for the genome, the population and the evolutionary algorithm
+```python
+params = {
+     'seed': 8188212,
+     'n_threads': 1,
```

Comment:
change to `n_processes` and add to `ea_params`

Reply:
Done. cef7b7a23477554b7cd4dfbba200927ed5fb897a